### DISCUSSION TITLE
Need Filter In List Page

### DISCUSSION THREAD

**Discussion by devimako** [area17/twill#2427] [Q&A]
<img width="1599" alt="image" src="https://github.com/area17/twill/assets/99637767/2150a5d9-6bb1-47fb-8c4e-2ea5988dd3b9">
i need a filter how could i get that option in my list page 

Thanks

**Comment by pauldwight** [comment]
@devimako There are 2 kinds of filters, Quick Filters (which are the ones like 'Mine', 'Draft' etc) and normal (from the Filter Dropdown). Have you tried to add filters by following the docs?

https://twillcms.com/docs/modules/table-builder.html#content-filters-1
